sale_timesheet_force_non_billable/views/task_timesheet_portal_templates.xml

```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
 
-    <template id="portal_my_task_timesheets" inherit_id="hr_timesheet.portal_my_task" name="Portal: My Task with billable Timesheets">
+    <!-- <template id="portal_my_task_timesheets" inherit_id="hr_timesheet.portal_my_task" name="Portal: My Task with billable Timesheets">
         <xpath expr="//div[@t-if='task.timesheet_ids']" position="attributes">
             <attribute name="t-if">task.timesheet_ids.filtered(lambda ts: ts.force_non_billable == false)</attribute>
         </xpath>
         <xpath expr="//tr[@t-foreach='task.timesheet_ids']" position="attributes">
             <attribute name="t-foreach">task.timesheet_ids.filtered(lambda ts: ts.force_non_billable == false)</attribute>
         </xpath>
-    </template>
+    </template> -->
```

Comment:
Why are we commenting this template?
I think there's probably a migration required here